feeds/tip/certificates/files/usr/bin/mount_certs

```diff
@@ -10,15 +10,25 @@ check_certificates
 mkdir -p /certificates /etc/ucentral/
 mtd=$(find_mtd_index certificates)
 
-if [ "$(head -c 4 /dev/mtd$mtd)" == "hsqs" ]; then
-	mount -t squashfs /dev/mtdblock$mtd /certificates
-else
+rootfstype=$(cat /proc/cmdline | sed 's/.*rootfstype=\([^ ]*\).*/\1/')
+case "$rootfstype" in
+squashfs)
```

Comment:
this will break 90% of boards

Reply:
Let me update the PR with updated code to cover this case.

Reply:
you will have to verify this on every supported board